core/engine.cpp

```diff
@@ -38,7 +38,7 @@
 
 void Engine::set_iterations_per_second(int p_ips) {
 
-	ips = p_ips;
+	ips = p_ips > 0 ? p_ips : 1;
```

Comment:
I would make it throw an error instead of overriding the value silently (users would wonder why it still runs at 1 FPS even though they asked for 0 or -42):
```
ERR_FAIL_COND(p_ips <= 0);
```